jibri/rootfs/etc/services.d/30-pulse/run

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/with-contenv bash
+#!/command/with-contenv bash
 
-HOME=/home/jibri
-exec s6-setuidgid jibri /bin/bash -c "exec /usr/bin/pulseaudio"
+HOME=/jibri
```

Comment:
Ah, so here is where we use /jibri. Why is /home/jibri not good?

Reply:
this is because we stopped creating jibri user, so initially this path would not exist. but now I'm creating /home/jibri and assigning the right permissions and therefore reverted this back to /home/jibri to minimize issues. PR will be updated soon..